shared/utils/trainingHelper.tsx

```diff
@@ -19,6 +19,7 @@ const appModes = [
   { label: 'Chapter 5', value: 'chapter5' },
   { label: 'Chapter 6', value: 'chapter6' },
   { label: 'Chapter 7', value: 'chapter7' },
+  { label: 'Chapter 8', value: 'chapter8' },
   { label: 'Chapter X', value: 'chapterX' },
```

Comment:
I'm unable to run with the `chapterX` lines in there. Can we remove them? 
![CleanShot 2025-01-02 at 13 06 21@2x](https://github.com/user-attachments/assets/b107329f-f7bd-468d-a13c-2812d4645ae4)

Reply:
yeah i'll remove in next push